src/classes/Commands.ts

```diff
@@ -916,15 +917,15 @@ export = class Commands {
             return;
         }
 
-        if (typeof params.buy === 'object') {
+        if (typeof params.buy === 'object' && params.buy !== null) {
             params.buy.keys = params.buy.keys || 0;
             params.buy.metal = params.buy.metal || 0;
 
             if (params.autoprice === undefined) {
                 params.autoprice = false;
             }
         }
-        if (typeof params.sell === 'object') {
+        if (typeof params.sell === 'object' && params.sell !== null) {
```

Comment:
I don't really like this, the bot does not support null prices.

I'll work on a fix and make a pull request.